scripts/signers_framework/signers_analysis.py

```diff
@@ -0,0 +1,102 @@
+from datetime import datetime, timezone
+import requests
+import pandas as pd
+from rich.console import Console
+from rich.pretty import pprint
+
+from brownie import Contract, web3
+from helpers.addresses import r
+
+console = Console()
+
+# NOTE: request info and fill it up over time depending on rotations
+replacements = {
+    "techops_multisig": {
+        "0x10C7a2ca16116E5C998Bfa3BC9CEF464475B18ff": "0x0a9af7FAba0d5DF7A8C881e1B9cd679ee07Af8A2",
+        "0xeE8b29AA52dD5fF2559da2C50b1887ADee257556": "0xfA5bb45895Cb3C0aE5B1583Fe068f009A48F0187",
+        "0xBB2281cA5B4d07263112604D1F182AD0Ab26a252": "0xE8eA1D8B3a5A4CEC7E94AE330fF18E82B5D22fA6",
+    }
+}
```

Comment:
i think these replacements should apply to the other msigs as well.

although not strictly the case, it should be safe enough to say those replacements are valid globally, not limited to a specific msig.

Reply:
good point, can add same rotation in other multisigs, where it is applicable. also i asked in ape channel for other members to commit those rotation, which may not be fully documented